lib/check.js

```diff
@@ -57,6 +57,7 @@ module.exports = function (options, callback) {
   }
 
   options.proxy = options.proxy || Conf.proxy;
+  options.proxy = options.proxy || process.env.https_proxy || process.env.HTTPS_PROXY;
```

Comment:
The reason is that lab will fail tests in that case, because Conf.proxy is always false. Lab ignores the last line in a conditional chain like that. In order to make lab happy any other way, we would have to pull in a .nsprc, modify it for that test, reload Conf, and change it back. While it looks strange, I thought this was a simpler solution that required fewer changes.